pages/api/auth/[...nextauth].js

```diff
@@ -29,6 +29,8 @@ export default NextAuth({
         },
     }),
     pages: {
+        // Redirect to the login.tsx page
+        signIn: "/login",
```

Comment:
Is this saying on sign in, we want to redirect them to login? Shouldn't we redirect them to home. Let me know if I'm misunderstanding.

Reply:
This is when we press sign-in on the index page. Not after they sign in. When signIn function is called on the index.tsx